modules/nw-ovn-kubernetes-migration-about.adoc

```diff
@@ -90,7 +95,12 @@ The difference in configuring an egress network policy, also known as an egress
 * Create an `EgressNetworkPolicy` object in a namespace
 |===
 
+[NOTE]
+====
+Because the name of an `EgressFirewall` object can only be set to `default`, after the migration all migrated `EgressNetworkPolicy` objects are named `default`, regardless of what the name was under OpenShift SDN.
```

Comment:
And if users rollback back the cluster, all the EgressNetworkPolicy objects will also use 'default' as the name, instead of their original name before the migration.